src/main/java/ttt/GamePlay.java

```diff
@@ -43,16 +43,14 @@ private void playGame() {
     private void placeMark(String input) {
         int choice = Integer.parseInt(input);
         gameEngine.play(choice);
-        if (gameEngine.getBoardStatus() == "mark placed") {
-            write(consoleBoard.update(gameEngine.nextPlayer.getMark(), choice));
-        } else {
-            write(consoleBoard.show());
-            invalidInput(gameEngine.getBoardStatus());
-        }
+        write(consoleBoard.update(gameEngine.currentPlayer.getMark(), choice));
```

Comment:
I'm not too keen on gameEngine exposing the currentPlayer, and having other public fields.

It would be good if you read about visibility modifiers, and think about the problems that having public fields may have.